### PR TITLE
Add invoice items

### PR DESCRIPTION
**Pull request by ppicatto**
| Q             | A
| ------------- | ---
| Branch	| master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR	| n/a

### CODE REVIEW

File: src/Api/InvoiceItems.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+namespace OpsWay\ZohoBooks\Api;
+
+class InvoiceItems extends BaseApi
+{
+    const API_PATH = 'items';
+    const API_KEY = 'item';
+
+    /**
+     * @link https://www.zoho.com/books/api/v3/settings/items/#mark-item-as-active.
```
Comment (phansys) [inline src/Api/InvoiceItems.php]: Please, use `@see` instead.

File: src/Api/InvoiceItems.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+     * @param string $itemId
+     *
+     * @return boolean
```
Comment (phansys) [inline src/Api/InvoiceItems.php]: Please, use `bool` instead.

**Review by phansys — CHANGES_REQUESTED** [review]
Status: Needs work.

**Review by phansys — APPROVED** [review]

**Comment by mrVrAlex** [comment]
Maybe make sense create new namespace "Settings"? Looks like in zoho documentation:
default

So it will be:
```php
namespace OpsWay\ZohoBooks\Api\Settings;

class InvoiceItems extends BaseApi {}
// Or even just Items for class name???
class Items extends BaseApi {}
```

What you think?

**Review by mrVrAlex — APPROVED** [review]

**Pull request merged**